packages/components/src/BidCreate.tsx

```diff
@@ -39,6 +39,7 @@ export function BidCreate({
                 name="amount"
                 id="amount"
                 mb={3}
+                min={0}
```

Comment:
<img width="373" alt="Screenshot 2021-12-13 at 11 37 53" src="https://user-images.githubusercontent.com/5102099/145797447-5f207145-b809-4d9b-8c20-d869d6bae3b9.png">

Not sure if I'm in the wrong branch but this doesn't seem to work for me @AshenI99